tests/routes/pro/get_booking_by_token_v2_test.py

```diff
@@ -89,7 +93,50 @@ def test_when_user_has_rights_and_regular_offer(self, app):
             "venueName": "Venue name",
         }
 
-    @pytest.mark.usefixtures("db_session")
+    def test_when_user_has_rights_and_booking_is_educational_validated_by_principal(self, app):
+        # Given
+        redactor = educational_factories.EducationalRedactorFactory(
+            civility="M.",
+            firstName="Jean",
+            lastName="Doudou",
+            email="jean.doux@example.com",
+        )
+        users_factories.AdminFactory(email="admin@example.com")
+        validated_eac_booking = bookings_factories.EducationalBookingFactory(
+            dateCreated=datetime.utcnow() - timedelta(days=5),
+            educationalBooking__educationalRedactor=redactor,
+            educationalBooking__status=EducationalBookingStatus.USED_BY_INSTITUTE,
+        )
+        url = f"/v2/bookings/token/{validated_eac_booking.token}"
+
+        # When
+        response = TestClient(app.test_client()).with_basic_auth("admin@example.com").get(url)
```

Comment:
```suggestion
        response = TestClient(app.test_client()).with_basic_auth(admin.email).get(url)
```
Comme cela pas besoin de définir l'email de l'utilisateur